src/components/landing/LandingPageOneToOne.module.css

```
@@ -0,0 +1,823 @@
+
+:global(#lookahead) .badge {
+  display: none;
+}
+
+.heroContent {
+  opacity: 0;
+  transform: translateY(30px);
+  animation: fadeInUp 1s ease forwards;
+}
+
+@keyframes fadeInUp {
```

Stylelint (17.12.0)

[error] Expected keyframe name "fadeInUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)